index.html

```diff
@@ -4,12 +4,223 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel='stylesheet' href='https://61f50f7fbe75680008babb7b--festive-mestorf-acecb0.netlify.app/style.css'/>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel='stylesheet' href='https://festive-mestorf-acecb0.netlify.app/style.css'/>
     <link rel='stylesheet' href='./style.css'/>
     <title>Documentation</title>
 </head>
 <body>
     
+    <header class='header-wrapper flex-horizontal-stretch flex-vertical-center letter-space-md padding-xxlg'>
+        <span class='font-size-xlg letter-space-lg'>DriftUI</span>
+        <nav>
+            <ul class='flex-horizontal-start font-size-md letters-space-sm'>
+                <li class='install-btn btn-filled padding-md border-radius-sm margin-right-xxlg text-capital'>setup</li>
+                <li class='doc-btn btn-filled padding-md border-radius-sm text-capital'>docs</li>
+            </ul>
+        </nav>
+    </header>
+
+    <div class='installation-div padding-xxlg flex-column flex-center'>
```

Comment:
use short meaningful class name other than this everything is fine :)

Reply:
okay ill update the class name.